course3/homework-vector/vector.cc

```diff
@@ -1,9 +1,53 @@
 #include <iostream>
 #include <cassert>
+#include <cmath>
 
 class Vector
 {
-    // TODO
+    private:
+        int private_x;
+        int private_y;
+
+    public:
+        // required constructors
+        Vector(int x, int y): private_x(x), private_y(y) {}
+        Vector() = default;
```

Comment:
The members will get initialized as pat of the default constructor.  An integer will be default initialized to 0.